README.md

```diff
@@ -1 +1,186 @@
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 Hello Microverse <a name="about-project"></a>
+
+I created my portfolio using HTML, CSS with Linters and Figma templates.
+
+## 🛠 Built With <a name="built-with"></a>
+1. HTML
+2. CSS
+3. Linters
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<!-- LIVE DEMO -->
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Added HTML
+- Added CSS
+- Added Linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+### Prerequisites
+
+1. A Browser (Preferably Google Chrome)
+2. A Code Editor
+3. Internet Connection
+4. Git
+
+<!-- SETUP -->
+### Setup
+
+Clone this repository to your desired folder using:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first.git
+```
+<!-- INSTALL -->
+
+### Install
+
+Install this project with the latest version of Git and a code editor.
+
+### Usage
+
+In order to run the project open index.html with live server.
+
+### Deployment
+
+You can deploy this project using:
+1.  A Code Editor
+2. Latest version of Git
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Cheptoo Patience**
+
+- GitHub: [@Cheptoo21](https://github.com/Cheptoo21)
+- Twitter: [@CheptooGyts](https://twitter.com/CheptooGyts)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/cheptoo-patience-73499015a)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **My Projects**
+- [ ] **My Languages Frameworks**
+- [ ] **My Contacts**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTION -->
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
[OPTIONAL] 
- [ ] Your `Contributing` section is missing the `issues` page, kindly add it to complete the `README` file, see the suggestion below:
```suggestion
## 🤝 Contributing <a name="contributing"></a>

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).

<p align="right">(<a href="#readme-top">back to top</a>)</p>
```